src/renderer/components/menu/menu.tsx

```diff
@@ -9,12 +9,13 @@ import debounce from "lodash/debounce";
 
 export const MenuContext = React.createContext<MenuContextValue>(null);
 export type MenuContextValue = Menu;
+export type MenuPositionSide = "inside" | "outside";
```

Comment:
I would like to see a comment explaining exactly what each of these means. I am having a really hard time trying to figure out what `{right: "outside", top: "inside"}` means.

Reply:
Imaging a box (parent element with borders). When you position a child element inside you can put it "in" or "out" of parent's boundaries. (if it wouldn't help I'll illustrate next time :)

Reply:
That sounds good, would you mind adding it as a comment in the code?

Reply:
Sure.